cmd/main.go

```diff
@@ -8,12 +8,25 @@ import (
 	"flag"
 	"log"
 	"mps-lookup/internal/proxy"
+	"os"
 )
 
 func main() {
+	
 	flag.Parse()
```

Comment:
remove since we aren't parsing command line flags

Reply:
done